build_macos.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+QTDIR=$1
+if [ "${QTDIR}" = "" ]; then
+  echo  "Usage: build.sh <qt-bin-directory> <optional version_suffix>"
+  exit 1
+fi
+
+PRO="kleiner-brauhelfer-2.pro"
+
+# comment out #define DEBUG true
+#sed -i '' -e '/^#define DEBUG true/ s/^#*/\/\/#/' ./source/src/definitionen.h
+
+${QTDIR}/qmake "${PRO}" || exit 1
+make clean && make || exit 1
+${QTDIR}/lupdate "${PRO}" || exit 1
+${QTDIR}/lrelease "${PRO}" || exit 1
+./deployment/macOS/deploy.sh ./bin/kleiner-brauhelfer.app "${QTDIR}" "$1"|| exit 1
```

Comment:
I think this should be `"$2"` for the version argument to the deploy script. Calling `build_macos.sh` without an optional version suffix leads to an error on my machine when `deploy.sh` tries to create the ZIP archive.

Reply:
Hi Michael, 

da hast du natürlich total recht. Habe ich gefixt.

Und da das Hobbybrauer-Forum, aus dem dies mehr oder weniger hervorgeht und die komplette Entickler-Crew deutschsprachig ist, machen wir das hier einfach auch so :-)

Reply:
OK. War aus Gewohnheit.